diff --git a/CMakeLists.txt b/CMakeLists.txt
index 75701ac..6575765 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,7 +20,7 @@
 
 cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
 # The version of CMake which is not compatible with the old CUDA CMake commands.
-set(CMAKE_VERSION_THRESHOLD "3.10.0")
+set(CMAKE_VERSION_THRESHOLD "3.15.0")
 
 if(${CMAKE_VERSION} VERSION_LESS ${CMAKE_VERSION_THRESHOLD})
   project(onnx2trt LANGUAGES CXX C)
@@ -89,6 +89,7 @@ set(API_TESTS_SOURCES
 
 set(HEADERS
   NvOnnxParser.h
+  NvOnnxParserTypedefs.h
 )
 
 if (NOT TARGET protobuf::libprotobuf)
